app/controllers/users_controller.rb

```diff
@@ -1,6 +1,35 @@
 class UsersController < ApplicationController
+# before_action :current_user?, only: [:show]
+
   def show
     @user = User.find(params[:id])
     authorize! :read, @user
   end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      session[:user_id] = @user.id
+      redirect_to user_path(@user.id), notice: "User created successfully."
+    else
+      redirect_to new_user_path, error: "Invalid Login Credentials"
```

Comment:
I think this should read "Invalid Credentials" like up above in the Admins#create method